en/Obsidian Sync/Obsidian Sync and third-party services.md

```diff
@@ -12,3 +12,22 @@ We recommend using a dedicated backup service that doesn't automatically update
 > Some cloud storage services, such as OneDrive, allow you to only download files when you use them and later remove them locally to free up space. Since the files are no longer available locally, Obsidian Sync believes they've been deleted and removes them from your remote vault.
 >
 > To use Obsidian Sync together with Files On-Demand and similar features, make sure to configure the service to always keep the files on the device.
+
+> [!QUESTION]- How do I move my vault out of iCloud Drive?
+> > [!INFO]- Move your vault out of iCloud Drive on macOS
+> > - Open the [[Vault Switcher]], right-click on the vault or tap the three dots, and select "Move vault."  
+>   > 
+> > If that doesn't work (e.g., if you are trying to move your vault across different drives):  
+> > 1. Quit the Obsidian app.  
+> > 2. Navigate to iCloud Drive → Obsidian in Finder  
+> > 3. Drag the vault folder to another non-cloud-sync'd location on your file system.  
+> > 4. Open the Obsidian app.  
+> > 5. If your vault doesn't open immediately, you may have to use the vault picker's "Open folder as vault" feature, then navigate to your vault's new location and open it.  
+>   
+> > [!INFO]- Move your vault out of iCloud Drive on iOS
+> > 1. Make a backup of your vault somewhere (just in case).  
+> > 2. On your Obsidian device, create a new, temporary vault, and make sure "Save in iCloud Drive" is **not** enabled. (This is to make sure the app container folder we'll use in the next step exists. You'll delete this fake vault in a moment.)  
+> > 3. Force quit the Obsidian app on all iOS devices and quit the app on any desktops. (This is to prevent Sync from doing anything while we move our files.)  
+> > 4. On your iOS device, go into the Files app, navigate to iCloud Drive → Obsidian, and long-press on your vault's folder. Select Move, and navigate to On My iPhone → Obsidian. (You should see the fake vault you created in step 2 here.) Press "Copy."  
+> > 5. Once the copying is finished: still in the Files app, navigate back to iCloud Drive → Obsidian. Delete your vault folder there.  
+> > 6. Open the Obsidian app. You should be presented with the vault picker and your vault should now have a vault icon instead of a cloud, indicating it is no longer on iCloud Drive.
```

Comment:
```suggestion
> > 8. Open the Obsidian app. 

In the vault switcher, your vault now has a vault icon instead of a cloud to indicate it's no longer synced to iCloud Drive. 
```